docs/extend/extend-cli/cli-installing-sample-plugin.md

```diff
@@ -1,36 +1,48 @@
 # Installing the sample plug-in
 
-Before you begin, [set up](cli-setting-up.md) your local environment to install a plug-in.
+This tutorial covers installing and running the [Zowe&trade; CLI sample plug-in](https://github.com/zowe/zowe-cli-sample-plugin/#zowe-cli-sample-plug-in), which adds a command to the CLI to list the contents of a directory on your computer.
 
-## Overview
-This tutorial covers installing and running this bundled Zowe&trade; CLI plugin as-is (without modification), which adds a command to the CLI that lists the contents of a directory on your computer.
+Before you begin, [set up](cli-setting-up.md) your local environment to install a plug-in.
 
 ## Installing the sample plug-in to Zowe CLI
 
-To begin, `cd` into your `zowe-tutorial` folder. (See [Initial setup](cli-setting-up.md#initial-setup) for instructions on creating the `zowe-tutorial` folder.)
+To install the Zowe CLI sample plug-in:
+
+1. Open a terminal and enter `cd zowe-tutorial` to change directory into your `zowe-tutorial` folder.
+
+    See [Initial setup](cli-setting-up.md#initial-setup) for instructions on creating the `zowe-tutorial` folder.
+
+2. Issue the following command to install the sample plug-in to Zowe CLI:
 
-Issue the following commands to install the sample plug-in to Zowe CLI:
+    ```
+    zowe plugins install ./zowe-cli-sample-plugin
+    ```
 
-`zowe plugins install ./zowe-cli-sample-plugin`
+    The Zowe CLI Sample Plug-in is installed.
 
 ## Viewing the installed plug-in
-Issue `zowe --help` in the command line to return information for the installed `zowe-cli-sample` command group:
+
+Open a terminal and issue the `zowe --help`command to return information for the installed `zowe-cli-sample` command group:
 
 ![Installed](../../images/guides/CLI/InstalledSample.png "Installed Sample Plugin")
 
 ## Using the installed plug-in
-To use the plug-in functionality, issue: `zowe zowe-cli-sample list directory-contents`:
+
+Open a terminal and issue the `zowe zowe-cli-sample list directory-contents` command:
 
 ![Output](../../images/guides/CLI/SampleOutput.png "Sample Plugin Output")
 
 ## Testing the installed plug-in
-To run automated tests against the plug-in, `cd` into your `zowe-tutorial/zowe-cli-sample-plugin` folder.
 
-Issue the following command:
+1. Open a terminal and enter `cd zowe-tutorial/zowe-cli-sample-plugin` to run automated tests against the plug-in.
 
-```
-npm run test
-```
+2. Issue the following command:
+
+    ```
+    npm run test
+    ```
+**[What happens when the command is issued? What/who performs the tests? How can the user see the results?]**
```

Comment:
@adam-wolfe please respond

Reply:
This command will run automated unit and system tests that the plug-in developer has defined in the __tests__ folder. 

After the tests are finished, the results of the test are printed to the terminal, showing the number of passed and failed tests, and identifying any tests that failed.